### PR TITLE
satellite-slave:  Validate schema of config settings.

### PR DESCRIPTION
**Pull request by mforsyth**

### CODE REVIEW

**Comment by wkf** [comment]
Done reviewing, just a few comments for you @mforsyth.

File: satellite-slave/src/satellite_slave/config.clj [hunk 1]
```diff
@@ -16,10 +16,28 @@
   (:require [cheshire.core :as cheshire]
             [clj-time.periodic :refer [periodic-seq]]
             [clj-time.core :as t]
+            [schema.core :as s]
             [satellite-slave.recipes]
             [satellite-slave.util :refer [every]])
   (:import (org.joda.time Period)))
 
+(defn joda-sequence?
+  "Returns true iff the first 3 elements of the sequence are joda times.
```
Comment (samccone) [inline satellite-slave/src/satellite_slave/config.clj]: iff => if

Comment (wkf) [inline satellite-slave/src/satellite_slave/config.clj]: iif means "if and only if"
-- 
William Farrell

Comment (samccone) [inline satellite-slave/src/satellite_slave/config.clj]: TIL IFF :goat:

**Comment by wkf** [comment]
Thanks @mforsyth. Looking sharp. :+1:

**Pull request merged**